javascript/ql/src/experimental/Security/CWE-614/InsecureCookie.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Failure to set secure cookies
+ * @description Insecure cookies may be sent in cleartext, which makes them vulnerable to
+ *              interception.
+ * @kind problem
+ * @problem.severity error
+ * @precision high
+ * @id js/insecure-cookie
+ * @tags security
+ *       external/cwe/cwe-614
+ */
+
+import javascript
+import InsecureCookie::Cookie
+
+from Cookie cookie
+where not cookie.isSecure()
+select "Cookie is added to response without the 'secure' flag being set to true (using " +
```

Comment:
```suggestion
select cookie, "Cookie is added to response without the 'secure' flag being set to true"
```

- This selects the right information for an alert query (mandatory)
- I dropped the `getKind()`, it is nice for testing, but seems redundant when looking at the concrete alerts (optional)
- You need to regenerate the expected outputs with this change (mandatory)